about.html

```diff
@@ -0,0 +1,103 @@
+  <p class="about-normal-text">Please contact us per Email for any further question about machin learnng online coures !</p>
+  <a id="about-link" href="#">machinelearning2022@gmail.com <img src="https://img.icons8.com/external-those-icons-lineal-those-icons/10/000000/external-right-arrows-those-icons-lineal-those-icons-1.png" alt="close"/></a>
+</section>
+<section class="logo-section">
+  <h2 class="lecturerTitle">Machine Learning Online Course 2022 Logo </h2>
+  <div class="red-line"></div>
+  <p class="about-normal-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent auctor dui eget purus interdum egestas. Aliquam quis dolor pharetra, blandit turpis non, rhoncus mauris. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Ut lobortis arcu vel porttitor maximus. Vestibulum non tellus vel ligula finibus posuere a a felis.</p>
+  <div class="logo-border"><img id="ml-logo" src="images/logo-large.png" alt="courseLogo"></div>
+</section>
+<section class="gallery">
+  <h2 class="lecturerTitle">See The past courses </h2>
+  <div class="red-line"></div>
+  <p class="about-normal-text" id="gallery-caption">Take a look at the last two machin learning workshop</p>
+  <div class="photoes">
+    <div class="photo-fram">
+    <img src="images/workshop1.jpg" class="photo" alt="workshop2021">
+      <div class="layer">
+        <h1 class="photo-caption">Ml workshop 2021</h1>
+      </div>
+    </div>
+    <div class="photo-fram">
+    <img src="images/workshop2.jpg" class="photo" alt="workshop2020">
+      <div class="layer">
+        <h1 class="photo-caption">Ml workshop 2022</h1>
+      </div>
+      </div>
+  </div>
+</section>
+<section class="mobile-partner">
+  <div class="mobile-partner-title">
+  <h2 class="programTitle">Partners</h2>
+      <div class="red-line"></div>
+  </div>
+      <div class="mobile-partner-list">
+      <a class="partner-item">Google</a>
+      <a class="partner-item">DataRobot</a>
+      <a class="partner-item">LogicMonitor</a>
+      </div>
+</section>
+<section id="darkLicense">
+  <img class="licence-logo" src="images/creative-commons-license-logo.jpg" alt="licence-logo"/>
+  <div>
+    <p class="licence-text">2015 Creative Commons <i class="fa fa-creative-commons" aria-hidden="true"></i></p>
+    <p class="licence-text">This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-nc/4.0/">Creative Commons Attribution-NonCommercial 4.0 International License</a></p>
+  </div>
+</section>
```

Comment:
- [ ] [OPTIONAL] Kindly ensure that the logo and the text are centered on the div and also remove the link attached to the text as it does not exist on the template. This will make your design look more like the provided template.
#### Your implementation
![2](https://user-images.githubusercontent.com/58692590/153420537-ad9dddff-17a7-4a5e-bc57-854c3086625e.png)
#### Provided Template
![22](https://user-images.githubusercontent.com/58692590/153420624-09db5a46-93a0-4e78-a137-a0712df1bacd.png)